src/pathfora.js

```diff
@@ -1165,29 +1150,19 @@
      * @param {object} config
      */
     constructWidgetActions: function (widget, config) {
-      var widgetOk = widget.querySelector('.pf-widget-ok');
-      var widgetAllCaptions;
-      var widgetFirstCaption;
-      var widgetCancel;
-      var widgetClose;
-      var widgetForm;
-      var widgetOnFormSubmit;
-      var widgetOnButtonClick;
-      var widgetOnModalClose;
-      var updateActionCookie;
-      var cancelShouldClose = true;
-      var i;
-      var j;
+      var widgetOk = widget.querySelector('.pf-widget-ok'),
+          widgetOnModalClose, updateActionCookie, widgetOnButtonClick;
```

Comment:
probably a nit but i wonder if we should make all variables with no value set consistently at the top of the var? some are above and some are below now.